k8s/data/broker.go

```diff
@@ -0,0 +1,32 @@
+package data
+
+import (
+	"k8s.io/client-go/1.4/pkg/api/unversioned"
+	"k8s.io/client-go/1.4/pkg/api/v1"
+)
+
+type BrokerState string
+
+const (
+	BrokerStatePending   BrokerState = "Pending"
+	BrokerStateAvailable BrokerState = "Available"
+	BrokerStateFailed    BrokerState = "Failed"
+)
+
```

Comment:
Alternatively, could we change `BrokerState` to `CatalogState` or even `BrokerCatalogState`?